app/controllers/houses_controller.rb

```diff
@@ -0,0 +1,16 @@
+class HousesController < ApplicationController
+
+  def index
+    @house = House.all.ordered
+  end
+
+
+  def show 
+    @house = House.find(params[:id])
+    @bookings_list= @house.bookings.sort_by(&:day) if params[:order]=="asc" || !params[:order]
+    @bookings_list= @house.bookings.sort_by(&:day).reverse if params[:order]=="desc"
```

Comment:
A `sort_by` helyett jobb lenne `order` -t használni. Így ez a kifejezés ActiveRecord oldalon tudna maradni, és gyorsabb lenne. Az `order` rögtön az SQL query-ben rendezi sorba a rekordokat, amíg a `sort_by` megvárja az SQL query eredményét, és utána, már a memóriába betöltött objektumokat, rendezi sorrendbe.
Így is működikk, de szebb lenne AR oldalon maradni.